test/acceptance/go.mod

```diff
@@ -3,12 +3,13 @@ module github.com/hashicorp/consul-helm/test/acceptance
 go 1.14
 
 require (
-	github.com/gruntwork-io/terratest v0.27.3
-	github.com/hashicorp/consul/api v1.5.0
-	github.com/hashicorp/consul/sdk v0.5.0
+	github.com/gruntwork-io/terratest v0.29.0
+	github.com/hashicorp/consul/api v1.6.0
+	github.com/hashicorp/consul/sdk v0.6.0
+	github.com/hashicorp/serf v0.9.3
```

Comment:
What required a dependency on `serf` out of curiosity?

Reply:
Good catch! *shakes fist at git rebase* I've rebased this branch with master and this retry keeps haunting me!

The answer is we never need a serf dependency 😄

Reply:
lol. yeah "rebase reintroducing things i fixed" has bitten me many a time.